DInvoker/examples/Example.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity >= 0.6.0;
+pragma AbiHeader expire;
+pragma AbiHeader time;
+pragma AbiHeader pubkey;
+import {DInvoker} from "DInvoker.sol";
+
+interface IResult {
+    function result(int sum) external;
+}
+
+contract CalculatorDebot {
+    function sum(int a, int b) public {
+        int c = a + b;
+        TvmCell response = tvm.encodeBody(IResult.result, c);
+        DInvoker.complete(msg.sender, response);
+    }
+}
+
+contract ExampleContract is IResult {
+
+    function testInvoke(address debot) public {
+        TvmCell request = tvm.encodeBody(CalculatorDebot.sum, 1, 2);
```

Comment:
Why we can't call other debot in regular way? Just like this:
```
function sum(int a, int b) public {
    IResult(msg.sender).result(a + b);
}
...
function testInvoke(address debot) public {
    CalculatorDebot(debot).sum(1, 2);
}
...
```
I think that debot engine can be smart enough to route messages between debots.

Reply:
I thought about that. Maybe you are right. But there are 2 reasons why i didn't use your way:
1) We need to implement mechanism of permissions where Debot Browser or User can allow or reject to invoke other debots.
2) If debot call will be executed onchain it will generate a successful transaction.